gemini/master.py

```diff
@@ -50,6 +50,20 @@ def join() -> str:
 
     return jsonify(response)
 
+@app.route('/v1/nodes/auth', methods=['POST'])
+def auth() -> None:
```

Comment:
We want to return a response here. If the password is incorrect, return a 403 or whatever the correct HTTP status code is. If it isn't return a 200 with the generated token in the body (I said otherwise in the issue but this sounds more correct).

Reply:
I have addressed this in new push.